src/gc-arena/src/collect_impl.rs

```diff
@@ -207,6 +207,27 @@ where
     }
 }
 
+#[cfg(feature = "hashbrown")]
+unsafe impl<K, V, S> Collect for hashbrown::HashMap<K, V, S>
+where
+    K: Eq + Hash + Collect,
```

Comment:
The `K: Eq + Hash` bound is unnecessarily restrictive (the bound could also be removed from `std::HashMap`'s impl).